components/MintPage/MintDay/Modals/MintMoreModal.tsx

```diff
@@ -103,9 +103,9 @@ const MintMoreModal: FC<MintMoreModalProps> = ({ isModalVisible, toggleIsVisible
               <></>
             )}
             {lockedCntOfCre8or ? (
```

Comment:
If unlocked = enter warehouse>
if all are locked = set up profile
I did thought like this before you....
lockedCntOfCre8or === 18 => set up profile
lockedCntOfCre8or !== 18 => enter ware house